src/components/MovieInfo/MovieInfo.js

```diff
@@ -10,7 +10,8 @@ const MovieInfo = ({ currentMovieInfo }) => {
       <article className='movie-stats'>
         <h2>{title}</h2>
         <h3>{average_rating}</h3>
-        // this is where we will add more info we want to display when we have it from the API fetch
+        <p>this is where we will add more info we want to display when we have it from the API fetch</p>
```

Comment:
Its hilarious that React hates commented out lines of code...love how we just turn them into <p> tags this mod!